docs/threat_model/mitigations.rst

```diff
@@ -182,15 +182,17 @@ Attacks on Network Infrastructure
 -  Landing Page is framed or unavailable
 -  Landing Page DNS leaks from SecureDrop/leaks-related subdomain
 -  Communications vulnerability in *Source* or *Journalist Interface*
--  DNS requests to news organization's subdomain for SecureDrop Landing Page, Freedom.press, torproject.org Tor activity, SD submissions may be correlated
+-  DNS requests to news organization's subdomain for SecureDrop Landing Page, 
+   Freedom.press, torproject.org Tor activity, SD submissions may be correlated
 -  SecureDrop.org is compromised
 -  User web traffic to SecureDrop Landing Page uses CDN and may be logged
 -  Tor network exploit
 -  apt server man-in-the-middle used to serve old or malicious packages
 -  SecureDrop apt servers are compromised, or apt server man-in-the middle attack injects malicious packages
 -  News Organization network is compromised
--  OSSEC and/or Journalist alert SMTP account credentials compromised
--  OSSEC and/or Journalist alert private key compromised
+-  *OSSEC Alert Public Key* and/or *Journalist Alert Public Key* SMTP account 
```

Comment:
This line refers to the email account credentials used to receive emails, not the keys used to encrypt them. So the original language is correct here, and should not be modified.